app/modules/conversations/conversations_router.py

```
@@ -213,6 +213,12 @@ async def post_message(
         # Check User-Agent header for local mode (same as regenerate_last_message)
         user_agent = http_request.headers.get("user-agent", "")
         local_mode = _is_vscode_extension_user_agent(user_agent)
+        logger.info(
```

Ruff (0.15.15)

[error] Too many arguments for `logging` format string

(PLE1205)